imagepicker/src/main/java/com/esafirm/imagepicker/features/ImagePickerActivity.java

```diff
@@ -117,6 +118,11 @@ private void setupView() {
             actionBar.setDisplayHomeAsUpEnabled(true);
             actionBar.setHomeAsUpIndicator(R.drawable.ic_arrow_back);
             actionBar.setDisplayShowTitleEnabled(true);
+
+            toolbar.setBackgroundColor(config.getActionBarColor());
```

Comment:
Should just leave it if it's not defined in `ImagePickerConfig` since it will take Application **Primary & Accent** colors